c_common/front_end_common_lib/local.mk

```diff
@@ -62,13 +62,13 @@ all: $(ALL_TARGETS)
 
 # All the c and h files built at the same time but individual rules needed for make chains
```

Comment:
think this file should be renamed to something akin to FrontEndCommon.mk

Reply:
Used front_end_common.mk to keep it distinct from the now defunct  FrontEndCommon.mk